BaseFilter/sections/specific.txt

```diff
@@ -1168,7 +1169,7 @@ mega4upload.com##div[class="mb-4 text-center"] > a > img
 trustedrevie.ws##.ads-area
 trustedrevie.ws##.col-md-12 > div:not(.rw-detail-companiews-area).see-all-info-area
 romsfun.com#?#h1.h3 ~ div[role="alert"]:has(a[href^="https://go.nordvpn.net/aff_c?offer_id="].btn)
-fplstatistics.co.uk##div[style^="position: relative; TOP:"] a[target="_blank"] > img
+fplstatistics.com##div[style^="position: relative; TOP:"] a[target="_blank"] > img
```

Comment:
If I'm not wrong, this rule (or rule above - `fplstatistics.com##div[style^="float:"] > a[target="_blank"] > img`) can be removed, because it hides the same element.